jest.config.js

```diff
@@ -0,0 +1,15 @@
+const { defaults: tsjPreset } = require('ts-jest/presets');
+
+module.exports = {
+  preset: 'ts-jest',
+  transform: {
+    // For transformations with .ts and .tsx files.
+    ...tsjPreset.transform,
+    // For transformations with .js and .jsx files.
+    '^.+\\.(js|jsx)$': 'babel-jest',
+  },
+  moduleNameMapper: {
+    '^@/(.*)$': '<rootDir>/$1',
+  },
```

Comment:
Since we're planning on removing aliasing support from create-wordpress-plugin anyway (we've already done this for -project and -theme) I'd recommend we avoid introducing it here only to have to remove it later.